src/style.css

```diff
@@ -1,3 +1,74 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+
+  --main-color: #000;
+  --additional-color: #00acdc;
+  --background-color: #fff;
+}
+
 body {
   margin: 0;
+  background-color: #eaeaea;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 100vw;
+  min-width: 1024px;
```

Comment:
it's a bad practice to hardcode the width to the blocks, as it will make this block not adaptive for other screen sizes